index.html

```diff
@@ -6,11 +6,83 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="./style.css">
 </head>
 
+<!-- Header space -->
+
 <body>
-    <!-- Add your markup here -->
+    
+    <header>
+
+<section class="bakery-icon">
+    <div class="cupcake-icon">
+        <img src="./design/cupcake_logo.png" alt="Bakery allusive icon">
+    </div>
+<br>
+<div>
+    <p class="welcome-msg">The best cakes in town delivered to your door</p>
+</div>
+
+</section>
+
+<!-- Menu / Navigation bar-->
+
+<nav>
+    <section class="mobile-nav">
+
+
+        <img src="./design/menu-hamburger.svg" class="hamburger-icon" onclick="navMenu()">
+          <div class="menu-nav" id="links">
+            
+            <ul>
+                <li><a href="#HOME">HOME</a></li>
+                <li><a href="#CAKES">CAKES</a></li>
+                <li><a href="#ORDERING">ORDERING</a></li>
+                <li><a href="#LESSONS">LESSONS</a></li>
+                 <li><a href="#ABOUT">ABOUT</a></li>
+            </ul>
+        </div>
```

Comment:
menu-nav seems to be your desktop nav - so it shouldn't live inside the `mobile-nav` section.

Also, the `manu-nav` `div` doesn't seem necessary - you could move the `menu-nav` class to the `ul` and remove the `div`. We want to avoid using more HTML than we need to.